driver/bpf/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM debian:unstable
+
+MAINTAINER Sysdig <support@sysdig.com>
+
+ENV SYSDIG_HOST_ROOT /host
+
+ENV AGENT_VERSION 0.1.1dev
+
+ENV SYSDIG_SRC /draios
+
+ENV HOME /root
+
+ENV CXX /usr/bin/g++
+
+# Based on the sysdig container, used for building eBPF probe
+
+RUN apt-get update \
+ && apt-get dist-upgrade -y \
+ && apt-get install -y --no-install-recommends \
+    apt-utils \
```

Comment:
Minor formatting issue: these packages are indented with spaces, the rest is indented with tabs.